Common/MathUtils/src/Chebyshev3D.cxx

```diff
@@ -507,15 +507,15 @@ Int_t Chebyshev3D::chebyshevFit(int dmOut)
 {
   // prepare paramaterization of 3D function for dmOut-th dimension
   int maxDim = 0;
-  for (int i = 0; i < 3; i++) {
-    if (maxDim < mNumberOfPoints[i]) {
-      maxDim = mNumberOfPoints[i];
+  for (int mNumberOfPoint : mNumberOfPoints) {
+    if (maxDim < mNumberOfPoint) {
```

Comment:
not really convinced ... in particular since the derived variable name is by far not optimal.
Also, the fact that the tool was able to make the conversion probably shows that there was no out-of-bound error initially?